Notebooks/OOPS_TMT_Statistical_testing.Rmd

```diff
@@ -123,52 +135,73 @@ volcano_plot <- function(lmResultsObj,
 ```
 
 ```{r}
-
-lmResults <- run_limma(data_to_model, design, coefficient='materialRBP:toxicToxic')
+limma_fit <- run_limma(data_to_model, design)
+lmResults <- summariseLmFit(limma_fit, coefficient='materialRBP:toxicToxic')
 
 volcano <- volcano_plot(lmResults,
                           adj.P.Val_threshold=0.1,
                           logFC_threshold=0.9,
                           xlabel='Effect of Toxic ASO on RNA binding (log2)')
 
 #define function build_results_list
-build_results_list <- function(data_to_model, design, results, volcano){
+build_results_list <- function(data_to_model, design, limma_fit, results, volcano){
```

Comment:
Adding `limma_fit` objec to the results list now that we have access to it outside the `run_limma` function.